WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/TotalRefundedCalculationUseCaseTests.swift

```diff
@@ -50,6 +49,6 @@ final class TotalRefundedCalculationUseCaseTests: XCTestCase {
         let totalRefunded = useCase.totalRefunded()
 
         // Then
-        XCTAssertEqual(totalRefunded, NSDecimalNumber(string: "-43427.9375229"))
+        XCTAssertEqual(totalRefunded, NSDecimalNumber(string: "-50867.7077171"))
```

Comment:
just curious, how did this number change from the changes?

Reply:
Previously, the test was not completely accurate, and it was actually passing for the case where the bug happens: if a refund was positive, the test would substract the amount refunded from the refund total, instead of adding it to it